src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task07.java

```diff
@@ -47,6 +49,17 @@ public static void main(String[] args) {
     static String generateTotalPriceList(int startNumberItems, double startPriceAllItems, int differentialNumberItems, int sizeTotalPrice) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        String totalPriceList = "";
+        double pricePerItem = startPriceAllItems / startNumberItems;
+        for (int i = 0, currentNumberItem = startNumberItems;
+             i < sizeTotalPrice;
+             i++, currentNumberItem += differentialNumberItems){
+            totalPriceList += currentNumberItem + " - "
+                    + NumberUtil.roundValueToTwoDigitsForMantissa(currentNumberItem * pricePerItem) ;
+            if (i != sizeTotalPrice - 1){
```

Comment:
"Violation: There isn't separating white spaces (OR line break) for curly brake
Solution: Add white spaces"